RFC-005-SWAP-Basic-Bitcoin.adoc

```diff
@@ -63,10 +64,10 @@ The parameters for the Bitcoin HTLC follow link:./RFC-003-SWAP-Basic.md#hash-tim
 |secret_hash
 |The hash of the `secret` (32 bytes for SHA-256)
 
-|redeem_identity
+|redeem_pubkeyhash
```

Comment:
I think we should stick here to `redeem_identity`. Even though the types are different to Ethereum, we still call it the same: 

https://github.com/comit-network/RFCs/blob/master/RFC-007-SWAP-Basic-Ether.md#contract

and 

https://github.com/comit-network/RFCs/blob/master/RFC-007-SWAP-Basic-Ether.md#parameters


```suggestion
| redeem_identity
```

Reply:
Well.. if looking closely, in the Ethereum RFC we use both: `redeem_identity` and `redeem_address`... so don't mind me :D